src/main/webapp/js/finscholar/init.js

```diff
@@ -30,22 +30,14 @@ const init = async () => {
   // Render the college page by loading in the data and rendering the 
   // associated soy template.
   try {
-    await renderCollege(GoogDom.getElement('college'));
+    const collegePage = new CollegePageView(GoogDom.getElement('college'));
+    await collegePage.renderCollege();
   } catch(err) {
     alert('Failed to initialize page. College page could not be rendered.');
```

Comment:
Instead of showing an alert, what do you think if we update the html of the element instead with the error message? Maybe we can do both.

Reply:
Maybe what would be better is to have a default template that displays the error message. So when an error occurs, we simply render that template.

Reply:
Ok so I'll create a new branch to work on this portion of error handling!